test/mocks/overrides/override_posts_feed_view_model.dart

```diff
@@ -39,8 +40,10 @@ final mockNonEmptyHomeViewModelOverride = [
 final mockLoadingHomeViewModelOverride = [
   postsFeedViewModelProvider.overrideWith(
     () => MockPostsFeedViewModel(
-      // Future.any([]) will never complete and simulate a loading state
-      build: () => Future.any([]),
+      build: () async {
+        await Future.delayed(Durations.long1);
+        return [];
+      },
```

Comment:
I am curious, why do you make it able to complete after 450 ms ?

Previously we had a never ending future

Reply:
This is also linked to the Flutter timers that you had to clear in `Display circular value on loading` (`test/views/pages/home/static_home_test.dart`).

With the never ending future, you are not forced to clear the flutter timers 😄

Reply:
To stay consistent with other tests, I now use `Durations.short1` (see b81aff5e) to avoid confusion about the exact delay of the future. We just want a short wait to try the loading.

I don't see how it is possible to test this without triggering an error if we don't clear the flutter timers.
If you manage to do it without changing the testing behavior, I would be happy if you can commit your solution on this branch.

Reply:
I see.

> I don't see how it is possible to test this without triggering an error if we don't clear the flutter timers. If you manage to do it without changing the testing behavior, I would be happy if you can commit your solution on this branch.

With the Future.wait([]), the test pass because we don't create a future, so none are pending when ending the test.

But if we keep the `Durations.short1`, yes we need to clear the flutter timers 👍🏼 

Thank you

Reply:
`Future.wait([])` is not a never ending future, right ? It will complete the next cycle.

Reply:
I think you confused `Future.wait([])` with `Future.any([])`.

Try the following in https://dartpad.dev:

```dart
void main() async {
  print("Starting.");
  await Future.wait([]); // completes instantly
  print("Done waiting on []");
  await Future.any([]); // never ending future
  print("This will not get logged.");
}
```